framework/kms-keyring.md

```diff
@@ -64,7 +64,7 @@ On keyring initialization, a keyring MAY define the following:
 
 ### Client Supplier
 
-A function that returns a KMS client that can make the following API calls in a particular AWS region:
+A function that maps a region string to a KMS client that can make the following API calls in the given AWS region:
```

Comment:
I think we want to be clear what the inputs/outputs are. The input is a string, the output is a KMS client.

Reply:
Can you help me understand in what way is this sentence unclear about that?

Reply:
The latest wording addresses my concern. I was just pointing out that using `maps` isn't as clear as explicitly stating what the input is and what the output is.